upgrade.md

```diff
@@ -17,6 +17,76 @@ The `storagePath` method of the `Illuminate\Contracts\Foundation\Application` in
 
     public function storagePath($path = '');
 
+<a name="mail"></a>
+### Mail
+
+<a name="symfony-mailer"></a>
+#### Symfony Mailer
+
+**Likelihood Of Impact: High**
+
+Swift Mailer is discontinued as of December 2021 and was therefore replaced with Symfony Mailer 6.0 (See https://symfony.com/blog/the-end-of-swiftmailer)
+Because of this SwiftMailer plugins and transports will no longer work with the new implementation.
+
+##### Updated Content Methods
+`send`, `html`, `text` & `plain` don't return the number of recipients anymore but an instance of `Illuminate\Mail\SentMessage` which contains the `Symfony\Component\Mailer\SentMessage` instance
+
+##### Renamed Swift methods to Symfony
+
+All Swift methods were renamed to Symfony. For example:
```

Comment:
May need a list of these, plus link to the Symfony Mailer documentation where they can read about the public methods available on a Symfony Mail message, etc.

Reply:
I looked at the PR for the framework and searched for all methods which where renamed.

Do you want to keep methods in the upgrade guide which are probably not used?
- Mailer::getSymfonyTransport()
- Mailer::setSymfonyTransport(TransportInterface $transport)
- MailManager::createSymfonyTransport($config)